### PR TITLE
Update `Sales Order` report footer

### PR DESCRIPTION
**Pull request by riadmen**
https://github.com/metasfresh/metasfresh/issues/19250

### CODE REVIEW

**Review by cristinamghita — APPROVED** [review]
lgtm

File: backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Docs_Inout_Details_Footer.sql [hunk 1]
```diff
@@ -1,77 +1,80 @@
...
+                NULL                 AS textcenter,
+                NULL                 AS language,
+                io.m_inout_id        AS m_inout_id,
+                'descr'              AS tag,
+                2                    AS pozition
+         FROM m_inout io
+         WHERE io.isActive = 'Y') footer
+         INNER JOIN m_inout io ON io.m_inout_id = footer.m_inout_id
+         INNER JOIN c_order o ON io.c_order_id = o.c_order_id
+         LEFT OUTER JOIN C_Incoterms inc ON o.c_incoterms_id = inc.c_incoterms_id
```
Comment (cristinamghita) [inline backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Docs_Inout_Details_Footer.sql:71]: pls, if you want to display the incoterm, take it from M_Inout, not from order.
And do not INNER JOIN with order. Might not have it

**Review by cristinamghita — CHANGES_REQUESTED** [review]
See my comment

File: backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Docs_Inout_Details_Footer.sql (hunk 1 above)
Comment (riadmen) [reply backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Docs_Inout_Details_Footer.sql]: correction done

**Review by cristinamghita — APPROVED** [review]
lgtm now

**Pull request merged**